### ISSUE TITLE
[a11y]: ai-chat does not announce main menu open and AI menu open for JAWS

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#913]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

When keyboard focus moves to interactive controls ,the screen reader fails to announce the current state expanded/collapsed

Recommended fix: Ensure that all interactive controls expose their current state programmatically, either through native HTML semantics or appropriate ARIA attributes
Here , use ARIA Expand /Collapse properties accordingly and indicate the Open /Closed state of the Show /Hide panel to Screen reader users.

User impact: Screen reader users are impacted because the role and state (e.g., checked, selected, expanded) of interactive controls are not announced on focus. This
prevents visually disabled users from understanding the condition of controls, leading to confusion and potential errors while interacting with the page.

<img width="402" height="301" alt="Image" src="https://github.com/user-attachments/assets/237a57b9-9781-49ec-88ee-b705bbde556b" />

### WCAG 2.1 Violation

WCAG Mapping: 4.1.2 Name, Role, Value

### Reproduction/example

n/a

### Steps to reproduce

1. open ai-chat
2. make sure to use JAWS
3. open main menu and AI menu
4. observe it is not properly announced, that either menu is opened

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
